charts/country-risk/values.yaml

```diff
@@ -0,0 +1,367 @@
+###############################################################
+# Copyright (c) 2022,2023 BMW Group AG
+# Copyright (c) 2022,2023 Contributors to the Eclipse Foundation
+#
+# See the NOTICE file(s) distributed with this work for additional
+# information regarding copyright ownership.
+#
+# This program and the accompanying materials are made available under the
+# terms of the Apache License, Version 2.0 which is available at
+# https://www.apache.org/licenses/LICENSE-2.0.
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+###############################################################
+
+
+country-risk-frontend:
+
+  replicaCount: 1
+
+  image:
+    registry: "ghcr.io"
+    # -- Name of the docker image
+    name: "catenax-ng/tx-vas-country-risk-frontend"
+    pullPolicy: Always
+    # -- Overrides the image tag whose default is the chart appVersion.
+    tag: ""
+
+  certificate:
+    # -- Hostname for the certificate
+    host: "frontend.countryrisk.example.net"
+
+  # -- Name of the backend service
+  appName: vas-country-risk-frontend
+
+  # -- List of secrets to be used
+  imagePullSecrets: []
+  # -- Annotations to be added to the running pod
+  podAnnotations: {}
+
+  # -- Configuration for security-related options of the running pod
+  podSecurityContext:
+    # -- Set the file system group ID for all containers in the pod
+    fsGroup: 2000
+
+  # Spring profiles under which the application should run
+  springProfiles:
+    - dev
```

Comment:
Is this a orphaned value? Did not find it in any Helm template file being used...

If it's orphaned, please remove section also in sub chart `values.yaml` files.

Reply:
done